command/output.go

```diff
@@ -98,7 +98,13 @@ func (c *OutputCommand) Run(args []string) int {
 		return 1
 	}
 
-	c.Ui.Output(v)
+	switch v.Type {
+	case "string":
+		c.Ui.Output(v.Value.(string))
+	default:
+		panic(fmt.Errorf("Unknown output type: %T", v.Value.(string)))
```

Comment:
I'd just `c.Ui.Error` and `return 1` here, but no big deal either way - this shouldn't be hit in normal operating conditions.